docs/install/sandboxes.rst

```diff
@@ -234,6 +234,87 @@ Notice that we can get the number of members of upstream clusters, number of req
 fulfilled by them, information about http ingress, and a plethora of other useful
 stats.
 
+Zipkin Tracing
+--------------
+
+The Zipkin tracing demonstrates Envoy's :ref:`request tracing <arch_overview_tracing>`
```

Comment:
can you reference this as an example of zipkin tracing configuration from https://lyft.github.io/envoy/docs/operations/faq/overview.html ?